doc/caching.rst

```diff
@@ -15,19 +15,26 @@ or simply undoing some changes within the same workspace.
 Configuration
 =============
 
-For now, Dune cache is an opt-in feature. There are three ways to enable it.
-Choose the one that is more convenient for you:
+.. TODO (ElectreAAS): update version number here when releasing
+Since %version%, Dune cache is now an opt-out feature: it is enabled by default.
```

Comment:
I think this can just be set to 3.17 already.

Reply:
Fixed now :)